README.md

```diff
@@ -64,9 +64,9 @@ The **[Catalog Specification](catalog-spec/)** specifies a structure to link var
 simple, flexible JSON file of links to Items, Catalogs or Collections that can be used in a variety of ways.
 
 The **[Collection Specification](collection-spec/)** provides additional information about a spatio-temporal collection of data.
-In the context of STAC it is most likely a collection of STAC Items that is made available by a data provider.
+In the context of STAC it is most likely a related set of STAC Items that is made available by a data provider.
```

Comment:
```suggestion
In the context of STAC it is most likely a related group of STAC Items that is made available by a data provider.
```

set could be interpreted as to imply uniqueness (which is true, but not relevant here)